src/parser/flags.ts

```diff
@@ -42,6 +43,20 @@ export const integer = build({
   },
 })
 
+export const directory = (opts: { exists?: boolean } & Partial<OptionFlag<string>> = {}): OptionFlag<string | undefined> => {
+  return build<string>({
+    ...opts,
+    parse: async (input: string) => opts.exists ? dirExists(input) : input,
```

Comment:
If `exists` is `false` this should at least validate the input is a valid directory path.

Reply:
can you explain what these validation would do (ex: what's valid vs. invalid?). Like certain characters that can't be used depending on the os?

Reply:
@mshanemc yep, exactly that.